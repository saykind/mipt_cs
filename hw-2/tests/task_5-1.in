13
1 2 3 4 3 2 1 0 -1 -2 -1 0 1
5
1.1 1.2 1.3 1.4 1.5
