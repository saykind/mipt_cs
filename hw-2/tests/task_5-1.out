13 6.500000